ci_framework/playbooks/02-infra.yml

```diff
@@ -4,9 +4,9 @@
     step: pre_infra
   ansible.builtin.import_playbook: ./hooks.yml
 
-- name: Infra playbook
-  hosts: "{{ cifmw_target_host | default('localhost') }}"
-  gather_facts: false
+- name: Prepare host virtualization
+  hosts: "{{ ('virthosts' in groups) | ternary('virthosts', cifmw_target_host | default('localhost') ) }}"
+  gather_facts: true
```

Comment:
As gathering facts is on by default - we don't need this line.

Reply:
Thanks. Let me fix it in the next patchset.

Reply:
we can be explicit - it doesn't hurt.